src/main/java/frc/robot/commands/AlignCommand.java

```diff
@@ -12,15 +13,39 @@
     private static final double Ki = 0;
     private static final double Kd = 0;
     private static double Kp = 1.0;
+    private boolean useLimelight;
+    private double turnToAngle;
     // Aka "Amy"
-    private final PIDController aimyAim = new PIDController(Kp, Ki, Kd);
+    private final PIDController pid = new PIDController(Kp, Ki, Kd);
     private final Limelight limelight;
     private final DriveSubsystem driveSubsystem;
+    private final AHRS navx;
 
     public AlignCommand(Limelight limelight, DriveSubsystem driveSubsystem) {
         this.limelight = limelight;
+        useLimelight = true;
         this.driveSubsystem = driveSubsystem;
+        this.navx = null;
 
+        PIDSetup();
+
+        addRequirements(driveSubsystem);
+    }
+
+    public AlignCommand(int turnToAngle, AHRS navx, DriveSubsystem driveSubsystem) {
```

Comment:
This would be a good place to create two subclass commands, `AlignWithLimelight` and `AlignWithGyro`.